### PR TITLE
added start md

### PR DESCRIPTION
**Pull request by Tiny-Pickles**
This is the start md file for ArcGIS @ebeshero @am0eba-byte @arrowarchive @Yuying-Jin

### CODE REVIEW

File: ArcGIS.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# ArgGIS #
```
Comment (ebeshero) [inline ArcGIS.md:1]: Why are you using a top-level heading for this?! Seriously?! NOOOO.

Just kidding. I approve.

**Review by ebeshero — APPROVED** [review]

File: ArcGIS.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+# twinkies for life! #
```
Comment (am0eba-byte) [inline ArcGIS.md:3]: ***NOT ENOUGH TWINKIES***

**Review by am0eba-byte — APPROVED** [review]
_never enough twinkies_

**Comment by ebeshero** [comment]
I disagree with the notion of not enough twinkies.

**Pull request merged**